ui/pages/snaps/snap-view/snap-settings.js

```diff
@@ -129,6 +130,15 @@ function SnapSettings({ snapId, initRemove, resetInitRemove }) {
     }
   };
 
+  const connectedTitle = () => {
```

Comment:
We still aren't showing icons or the proper names for Snaps in this list.

![image](https://github.com/user-attachments/assets/c159e484-3aac-4384-a376-1765f5759a3e)

Reply:
Oh I didn't realize we wanted to show the snap name, I thought it was supposed to be just the id? As far as the icon, is this supposed to be a snap provided icon and fallback is the snaps logo?

Reply:
Lets just use `SnapIcon` which has a fallback already!

Reply:
Done!